autogen/configureModel.m

```diff
@@ -22,7 +22,7 @@
 config = jsondecode(fileread(configPath));
 
 mp.modelName = config.devices(1).name;
-
+mp.useAvalonInterface = lower(config.system.processing) == "channel";
```

Comment:
If we can come up with a better name than `mp.useAvalonInterface`, I'm all for changing that right now. Maybe `mp.channelBasedProcessing` or `mp.isChannelBased`